### PR TITLE
PLT-3364 Fix panic when load config fails

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

File: mattermost.go [hunk 1]
```diff
@@ -71,7 +71,7 @@ var flagRunCmds bool
 func doLoadConfig(filename string) (err string) {
 	defer func() {
 		if r := recover(); r != nil {
-			err = r.(string)
+			err = fmt.Sprintf("Error loding config")
```
Comment (jwilander) [inline mattermost.go]: Can you make this `fmt.Sprintf("Error loding config, err=%v", r)`? That should work to print out an error without having to type convert manually

**Comment by enahum** [comment]
@jwilander done

**Comment by coreyhulen** [comment]
+1

**Comment by crspeller** [comment]
+1

**Pull request merged**